pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -2199,3 +2204,16 @@ def _check_unnecessary_list_index_lookup(
                 args=(node.target.elts[1].name,),
                 confidence=HIGH,
             )
+
+    def _enumerate_with_start(self, node: nodes.For | nodes.Comprehension) -> bool:
```

Comment:
This works, but what if you just replace `or not node.iter.args` above with `or len(node.iter.args) != 1`? 🤔

Reply:
given the more complicated logic, this isn't applicable